apps/omg_watcher_rpc/priv/swagger/informational_api_specs/transaction/paths.yaml

```diff
@@ -47,6 +47,21 @@ transaction.get:
       500:
         $ref: '../../../../../omg_child_chain_rpc/priv/swagger/shared/responses.yaml#/InternalServerError'
 
+transaction.submit_typed:
+  post:
+    tags:
+      - Transaction
```

Comment:
I considered it a good idea on the first spot, but now I'm not so sure. Here is why:

> YAML is for us a tool to not having to write JSON :) 

and we're using just enough syntax to accomplish the job. If I'd compact yaml in the places I've touched it will be inconsistent to the rest. 

I see consistency as a value because everybody without knowing much about yaml & open API spec can jump up, learn from examples and describe some other endpoint. 

I'm not going to change this then...